packages/react-reconciler/src/ReactFiberPendingWork.js

```diff
@@ -0,0 +1,243 @@
+/**
```

Comment:
It's a bit confusing that it refers to "Work" which is not really work like beginWork but levels. `ReactFiberPendingWorkPriority`?

I don't like this file as expected. I don't think this is needed and that we can just infer this but let's come back to that.

What I like the least is that this slows down everything whether you use this feature or if it's even enabled. That's the kind of thing that makes me not want to land it even temporarily.